copy/copy_linux.go

```diff
@@ -52,6 +52,21 @@ func (c *copier) copyFileInfo(fi os.FileInfo, name string) error {
 	return nil
 }
 
+func copyFile(source, target string) error {
```

Comment:
you could call this `copyFileNaive` in `copy.go` and call it as a fallback from darwin.

Reply:
But this function is not naive in linux. It does copy_file_range on linux.